ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh

```diff
@@ -0,0 +1,69 @@
+    sleep 2m
+    #Samples operator queries istag per 15 mins, it's too slow to update istags. But deleted imagestreams could be re-created directly.
+    echo "#To speed up the two imagestreams updating manaully"
+    oc delete is ruby httpd -n openshift
+}
+
+function mirror_tag_images () {
+    echo "quay.io/openshift-qe-optional-operators/ocp4-index:latest=MIRROR_REGISTRY_PLACEHOLDER/openshift-qe-optional-operators/ocp4-index:latest
+registry.redhat.io/ubi8/ruby-30:latest=MIRROR_REGISTRY_PLACEHOLDER/ubi8/ruby-30:latest
+registry.redhat.io/ubi8/ruby-27:latest=MIRROR_REGISTRY_PLACEHOLDER/ubi8/ruby-27:latest
+registry.redhat.io/ubi7/ruby-27:latest=MIRROR_REGISTRY_PLACEHOLDER/ubi7/ruby-27:latest
+registry.redhat.io/rhscl/ruby-25-rhel7:latest=MIRROR_REGISTRY_PLACEHOLDER/rhscl/ruby-25-rhel7:latest
+registry.redhat.io/rhscl/mysql-80-rhel7:latest=MIRROR_REGISTRY_PLACEHOLDER/rhscl/mysql-80-rhel7:latest
+registry.redhat.io/rhel8/mysql-80:latest=MIRROR_REGISTRY_PLACEHOLDER/rhel8/mysql-80:latest
+registry.redhat.io/rhel8/httpd-24:latest=MIRROR_REGISTRY_PLACEHOLDER/rhel8/httpd-24:latest
+quay.io/openshifttest/osus-graph-data-container:latest=MIRROR_REGISTRY_PLACEHOLDER/openshifttest/osus-graph-data-container:latest
+" > /tmp/tag_images_list
+
+    run_command "oc extract secret/pull-secret -n openshift-config --confirm --to /tmp"; ret=$?
+    if [[ $ret -eq 0 ]]; then
+        echo "Get the cluster global pull secret succeessfully."
+    else
+        echo "!!! Fail to get the cluster global pull secret"
+        return 1
+    fi
+
+    sed -i "s/MIRROR_REGISTRY_PLACEHOLDER/${REGISTRY_HOST}:5000/g" "/tmp/tag_images_list"
+    run_command "cat /tmp/tag_images_list"
+    run_command "oc image mirror -f \"/tmp/tag_images_list\"  --insecure=true -a \"/tmp/.dockerconfigjson\" --skip-missing=true --skip-verification=true --keep-manifest-list=true --filter-by-os='.*'"; ret=$?
+    if [[ $ret -eq 0 ]]; then
+        echo "Mirror tag images to the Proxy registry succeessfully."
+    else
+        echo "!!! Fail to mirror tag images to the Proxy registry"
+        run_command "cat /tmp/.dockerconfigjson"
+        return 1
+    fi
+}
+
+run_command "oc get config.samples.operator.openshift.io cluster"; ret=$?
+if [[ $ret -eq 0 ]]; then
+    echo "The Sample operator installed in the cluster, continue..."
```

Comment:
Should we add config_samples_operator here?

Reply:
No, just judge if the Sample operator installed in the cluster, if not, return.

Reply:
To solve: https://github.com/openshift/release/pull/30628#issuecomment-1191033620